openedx/core/djangoapps/credit/tests/test_tasks.py

```diff
@@ -257,15 +257,15 @@ def test_icrv_requirement_ordering(self):
         on_course_publish(self.course.id)
         requirements = get_credit_requirements(self.course.id, namespace="reverification")
         self.assertEqual(len(requirements), 4)
-        self.assertEqual(requirements[0]["display_name"], "Midterm Start Date")
-        self.assertEqual(requirements[1]["display_name"], "Midterm Start Date")
-        self.assertEqual(requirements[2]["display_name"], "Midterm B")
-        self.assertEqual(requirements[3]["display_name"], "Midterm A")
+        self.assertEqual(requirements[0]["display_name"], "Midterm A")
```

Comment:
Please add  comment about sorting, since we are now primarily sorting on `start_date` and `display_name` if `start_date` is present otherwise we are just sorting on `display_name`.